data/tutor.yaml

```diff
@@ -251,12 +253,17 @@
       CPSC314,
       CPSC317,
       CPSC320,
-      CPSC418
+      CPSC418,
+      APSC 160,
```

Comment:
in APSC 160, there is space between the department code and the course number, but for CPSC courses, like CPSC418, there is no space. Should we standardize the format for consistency, if there's no specific reason for the difference?